constraints.md

```diff
@@ -189,7 +79,7 @@ SELECT * FROM users;
 
 ## 唯一约束
 
-在 TiDB 中，默认会对唯一约束进行惰性检查。通过直到事务提交时才进行批量检查，TiDB 能够减少网络通信开销。例如：
+在 TiDB 的乐观事务中，默认会对唯一约束进行[惰性检查](/transaction-overview.md#惰性检查)。通过在事务提交时再进行批量检查，TiDB 能够减少网络开销、提升性能。例如：
```

Comment:
This is different from English ``` Depending on the transaction mode and the value of `tidb_constraint_check_in_place`, TiDB might check `UNIQUE` constraints [lazily](/transaction-overview.md#lazy-check-of-constraints). This helps improve performance by batching network access. ```. Is this expected?

Reply:
Yes. This is to keep it consistent with `dev`, confirmed in https://github.com/pingcap/docs-cn/pull/4263#discussion_r484623031